api/origin_trials_api.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright 2023 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from framework import basehandlers
+import requests
+import settings
+
+
+class OriginTrialsAPI(basehandlers.APIHandler):
+
+  def do_get(self, **kwargs):
+    """Get a list of all origin trials."""
+    key = settings.OT_API_KEY
+    # Return an empty list if no API key is found.
+    if key == None:
+      return []
+
+    response = requests.get(
```

Comment:
If the key exists, we should wrap the request and the parsing in a try-except. In the event something goes wrong, log the exception internally and send back a generic error for the frontend to display.

Reply:
Yes, agreed - I've added a try/except block with the approach you mentioned 👍